src/Akeneo/Bundle/ElasticsearchBundle/Cursor/FromBoundedCursor.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace Akeneo\Bundle\ElasticsearchBundle\Cursor;
+
+use Akeneo\Bundle\ElasticsearchBundle\Client;
+use Akeneo\Component\StorageUtils\Cursor\CursorInterface;
+use Akeneo\Component\StorageUtils\Repository\CursorableRepositoryInterface;
+
+/**
+ * Bounded cursor to iterate over items where a start and a limit are defined.
+ * Internally, this is implemented with the from/size pagination.
+ * {@see https://www.elastic.co/guide/en/elasticsearch/reference/5.x/search-request-from-size.html}
+ *
+ * @author    Julien Janvier <jjanvier@akeneo.com>
+ * @author    Marie Bochu <marie.bochu@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class FromBoundedCursor extends AbstractCursor implements CursorInterface
```

Comment:
Ok, maybe my comment is distorted by the API. 
Btw, ES doc describes "from" as it: "The from parameter defines the offset" that's why I said that.